<?xml version="1.0"?>

<launch>
   <arg name="model" default="$(find hector_quadrotor_description)/urdf/quadrotor_hokuyo_utm30lx.gazebo.xacro" />

   <group ns="uav1">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav1" />
       <arg name="tf_prefix" value="uav1" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="-1.0" />
     </include>
   </group>

   <group ns="uav2">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav2" />
       <arg name="tf_prefix" value="uav2" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="1.0" />
     </include>
   </group>

  <group ns="uav3">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav3" />
       <arg name="tf_prefix" value="uav3" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="2.0" />
     </include>
   </group>

   <group ns="uav4">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav4" />
       <arg name="tf_prefix" value="uav5" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="-2.0" />
     </include>
   </group>

  <group ns="uav5">
     <include file="$(find hector_quadrotor_gazebo)/launch/spawn_quadrotor.launch">
       <arg name="name" value="uav5" />
       <arg name="tf_prefix" value="uav5" />
       <arg name="model" value="$(arg model)" />
       <arg name="y" value="0.0" />
     </include>
   </group>

  <node pkg="hector_quadrotor_gazebo" type="opencv-camera.py" name="start_camera_" output="screen"/>
  <node pkg="hector_quadrotor_gazebo" type="opencv-camera2.py" name="start_camera__" output="screen"/>
  <node pkg="hector_quadrotor_gazebo" type="opencv-camera3.py" name="start_camera___" output="screen"/>
  <node pkg="hector_quadrotor_gazebo" type="opencv-camera4.py" name="start_camera____" output="screen"/>
  <node pkg="hector_quadrotor_gazebo" type="opencv-camera5.py" name="start_camera_____" output="screen"/>
</launch>

